x-pack/libbeat/management/error.go

```diff
@@ -21,58 +17,13 @@ type ErrorType string
 // ConfigError is the type of error send when an unpack or a blacklist happen.
 var ConfigError = ErrorType("CONFIG")
 
-// ErrorEvent is the event type when an error happen.
-var ErrorEvent = api.EventType("ERROR")
-
 // Error is a config error to be reported to kibana.
 type Error struct {
```

Comment:
i winder if we need to keep it here, we are coupling error to type and UUID but Error method just calls `Error` of nested one and we dont expose coupled information anywhere.

Reply:
Good point. It looks that the only reason for having this type was json marshalling/unmarshalling. Might indeed be obsolete as well.